app/models/picture.rb

```diff
@@ -0,0 +1,12 @@
+class Picture < ActiveRecord::Base
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+
+  has_attached_file :image,
+                    styles: {medium: "300x300>", thumb: "100x100>"},
+                    :storage => :s3,
+                    :bucket  => ENV['S3_BUCKET_NAME'],
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)